### DISCUSSION TITLE
Analytics output Item price - adding 6 zero after value

### DISCUSSION THREAD

**Discussion by Shperung** [invertase/react-native-firebase#6084] [General]
When I use https://rnfirebase.io/reference/analytics#logAddToCart. I send Item price 19. But in analitics i see 19000000. This is bug or feature? How i need send price?  
![image](https://user-images.githubusercontent.com/9537441/154506207-6cf465a4-727d-4a19-89db-e95d3e94e148.png)

**Comment by Shperung** [comment]
versions 
`    "@react-native-firebase/analytics": "14.5.0",
    "@react-native-firebase/app": "14.5.0",`

**Reply by mikehardy** [reply]
I think it would help to include a snippet of code, and try various things then report what results were: https://stackoverflow.com/help/how-to-ask

More importantly than that, it appears you did not search the issues list, check out this comment, it is exactly what you are experiencing I think https://github.com/invertase/react-native-firebase/issues/4578#issuecomment-771703420

**Comment by Shperung** [comment]
My snippet of code
![image](https://user-images.githubusercontent.com/9537441/154513728-02e45353-64f4-4a20-8b0c-c859952992ef.png)

**Reply by Shperung** [reply]
[in comment ](https://github.com/invertase/react-native-firebase/issues/4578#issuecomment-771703420) i don't find answer

**Reply by mikehardy** [reply]
The answer is: that's the way firebase-ios-sdk behaves, we just wrap firebase-ios-sdk, if that's the way the underlying SDK does things, that is just the way things are - that's what I understood from the comments at least. Have you looked in firebase-ios-sdk repository?

**Reply by Shperung** [reply]
@mikehardy Thanks i understood. I don't looked in firebase-ios-sdk repository. Only used @react-native-firebase/analytics

**Reply by mikehardy** [reply]
Yep - important to know that `@react-native-firebase/analytics` is just javascript bindings on `firebase-ios-sdk/analytics` and `firebase-android-sdk/analytics` and from the comment I linked, it appears `firebase-ios-sdk/analytics` has this behavior, thus, we have this behavior
